logging/src/main/kotlin/com/ritense/logging/web/rest/LoggingEventManagementResource.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.logging.web.rest
+
+import com.ritense.logging.service.LoggingEventService
+import com.ritense.logging.web.rest.dto.LoggingEventResponse
+import com.ritense.logging.web.rest.dto.LoggingEventSearchRequest
+import com.ritense.valtimo.contract.annotation.SkipComponentScan
+import com.ritense.valtimo.contract.domain.ValtimoMediaType.APPLICATION_JSON_UTF8_VALUE
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.PageImpl
+import org.springframework.data.domain.Pageable
+import org.springframework.data.domain.Sort
+import org.springframework.data.web.PageableDefault
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@SkipComponentScan
+@RequestMapping("/api/management", produces = [APPLICATION_JSON_UTF8_VALUE])
+class LoggingEventManagementResource(
+    private val loggingEventService: LoggingEventService,
+) {
+
+    @PostMapping("/v1/logging")
+    fun searchLoggingEvents(
+        searchRequest: LoggingEventSearchRequest,
+        @PageableDefault(sort = ["timestamp"], direction = Sort.Direction.DESC) pageable: Pageable,
```

Comment:
Did we want to base this on last id in the list to prevent weird results when viewing the logs? The pages will change fast.

Reply:
This can also be achieved with the `beforeTimestamp`. One of the tests shows an example of this

Reply:
We'll see how well this will work when we have a front-end. approving this for now.